### ISSUE TITLE
"npm run def" is broken typescript definition

### ISSUE THREAD

**Issue by Vorobeyko** [Codeception/CodeceptJS#2024]
#### What are you trying to achieve?
1. Add new method for helper
2. Run `npm run docs`
3. Run `npm run def`

#### What do you get instead?
Changes for typings!
And broken unit tests

and error

```bash
ERROR: Unable to find the source file or directory /home/avvorobey/github/TinkoffCreditSystem/CodeceptJS/lib/Helper.js
[TSD-JSDoc] Failed to find parent of doclet 'module.exports' using memberof 'module', this is likely due to invalid JSDoc.
[TSD-JSDoc] Failed to find parent of doclet 'container.mocha' using memberof 'container', this is likely due to invalid JSDoc.
[TSD-JSDoc] Failed to find parent of doclet 'module.exports' using memberof 'module', this is likely due to invalid JSDoc.
[TSD-JSDoc] Failed to find parent of doclet 'context.Scenario' using memberof 'context', this is likely due to invalid JSDoc.
[TSD-JSDoc] Unable to resolve type for an unnamed item, this is likely due to invalid JSDoc. Often this is caused by invalid JSDoc on a parameter. Defaulting to `any`.
[TSD-JSDoc] Unable to resolve type for an unnamed item, this is likely due to invalid JSDoc. Often this is caused by invalid JSDoc on a parameter. Defaulting to `any`.
[TSD-JSDoc] Unable to resolve type for an unnamed item, this is likely due to invalid JSDoc. Often this is caused by invalid JSDoc on a parameter. Defaulting to `any`.
```

### Details

* CodeceptJS version: master
* NodeJS Version: v10.17.0
* Operating System: linux
* Protractor || WebDriverIO || Nightmare version (if related)
* Configuration file:

**Comment by Vorobeyko** [comment]
fixed in release 2.4! :tada:

**Comment by Georgegriff** [comment]
Seeing similar issues again in 3.x Does anyone have any knowledge here? Posted in dev channel to discuss
